tools/scripts/diff-sizes.mjs

```diff
@@ -136,7 +147,39 @@ ${generateDiffRows(assetSizes, 'standard')}
 ${generateDiffRows(assetSizes, 'polyfills')}
 ${generateDiffRows(assetSizes, 'worker')}
 
-<sub>This report only accounts for the minified loaders. Un-minified and async-loaded assets are not included.</sub>
+<details>
+<summary>Other Standard Assets</summary>
+
+## Released Assets
+
+| Asset Name | Asset Size |
```

Comment:
Can the "previous" and "new" sizes be in the same table like it is for the loaders? (Calculating diff isn't necessary.) Make it easier to read.

Reply:
Not without a lot of extra work to find match the asset names across releases